.eslintrc.cjs

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2024: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    "plugin:react-hooks/recommended",
+    'airbnb-typescript',
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended',
+    'plugin:cypress/recommended',
+  ],
+  overrides: [
+    {
+      'files': ['**/*.spec.jsx'],
+      'rules': {
+        'react/jsx-filename-extension': ['off'],
+      }
+    }
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
```

Comment:
The `ecmaVersion` is set to 12, which corresponds to ECMAScript 2021. Ensure this matches your intended JavaScript version, especially since `es2024` was specified in the `env` section.